code/modules/surgery/organs/subtypes/skrell.dm

```diff
@@ -9,42 +9,34 @@
 	icon = 'icons/obj/species_organs/skrell.dmi'
 	icon_state = "skrell_headpocket"
 	origin_tech = "biotech=2"
-	w_class = WEIGHT_CLASS_TINY
+	w_class = WEIGHT_CLASS_SMALL
 	parent_organ = "head"
 	slot = "headpocket"
 	actions_types = list(/datum/action/item_action/organ_action/toggle)
-	var/obj/item/storage/internal/pocket
-
-/obj/item/organ/internal/headpocket/New()
-	pocket = new /obj/item/storage/internal(src)
-	pocket.storage_slots = 1
-	// Fit only pocket sized items
-	pocket.max_w_class = WEIGHT_CLASS_SMALL
-	pocket.max_combined_w_class = 2
-	..()
+	var/obj/item/held_item
 
 /obj/item/organ/internal/headpocket/Destroy()
-	QDEL_NULL(pocket)
-	return ..()
-
-/obj/item/organ/internal/headpocket/insert(mob/living/carbon/M, special = FALSE, dont_remove_slot = 0)
-	. = ..()
-	pocket.master_item = M // So headpockets work with adjacency checks
-
-/obj/item/organ/internal/headpocket/remove(mob/living/carbon/M, special = 0)
-	pocket.master_item = src
+	QDEL_NULL(held_item)
 	return ..()
 
 /obj/item/organ/internal/headpocket/on_life()
 	..()
 	var/obj/item/organ/external/head/head = owner.get_organ("head")
-	if(pocket.contents.len && !findtextEx(head.h_style, "Tentacles"))
-		owner.visible_message("<span class='notice'>Something falls from [owner]'s head!</span>",
-													"<span class='notice'>Something falls from your head!</span>")
+	if(held_item && !findtextEx(head.h_style, "Tentacles"))
+		owner.visible_message("<span class='notice'>[held_item] falls from [owner]'s [src]!</span>", "<span class='notice'>[held_item] falls from your [src]!</span>")
```

Comment:
Needs to be changed to `falls from your [name]!` otherwise you get `falls from your the headpocket!`